kie-wb-tests/kie-wb-tests-gui/src/test/java/org/kie/wb/selenium/ui/LoadAllPerspectivesIntegrationTest.java

```diff
@@ -31,37 +30,26 @@
  */
 public class LoadAllPerspectivesIntegrationTest extends KieSeleniumTest {
 
-    private static final Logger LOG = LoggerFactory.getLogger( LoadAllPerspectivesIntegrationTest.class );
+    private static final Logger LOG = LoggerFactory.getLogger(LoadAllPerspectivesIntegrationTest.class);
 
     @Test
     public void allPerspectivesCanBeLoaded() {
         login.getLoginPage();
         HomePerspective home = login.loginDefaultUser();
 
-        for ( Persp<?> p : Persp.getAllPerspectives() ) {
-            if ( perspectiveInCurrentWebApp( p ) ) {
-
-                Persp<?> _p = p;
-                LOG.info( "Checking perspective '" + p.getName() + ".." );
-                AbstractPerspective perspective = home.getNavbar().navigateTo( _p );
-
-                //ProjectAuthoring will direct to ProjectLibrary if there are no Projects in the workbench
```

Comment:
Just removed the inner if and reformatted the code. The if was necessary when we had the dichotomy Authoring/Library (navigating to authoring with no project redirected to library). Since Authoring has been removed and the "Project Authoring" menu item now always takes you to library perspective, we no longer need this "skip authoring" workaround